README.md

```diff
@@ -44,6 +44,11 @@ For Kubernetes 1.14/1.15 users
 TAG=v0.4.0
 kubectl apply -f ./install/$TAG/kfserving.yaml --validate=false
 ```
+KFServing can also be installed standalone in `kubeflow` namespace.
+```
+TAG=v0.4.0
+kubectl kustomize ./config/overlays/kubeflow | sed s/:latest/:$TAG/ | kubectl apply -f -
```

Comment:
Can we generate and materialize the kubeflow kfserving manifest yaml and check into the install directory as we change the kustomize config all the time ?

Reply:
Would you be more specific, please? I would like to know what do you want to do.

Reply:
@jazzsir we changed kfserving manifests quite a bit after 0.4.0 in master, so it is very likely no longer working with 0.4.0 release if we run the above command and generate from master. So I'd prefer add the above command to the [install script](https://github.com/kubeflow/kfserving/blob/master/hack/generate-install.sh#L41) and we can use that to generate the kubeflow manifest yaml for KFServing when we release 0.5.0.

```
INSTALL_DIR=./install/$TAG
INSTALL_KUBEFLOW_PATH=$INSTALL_DIR/kfserving_kubflow.yaml
kustomize build config/overlays/kubeflow | sed s/:latest/:$TAG/ > $INSTALL_KUBEFLOW_PATH
```

Reply:
@yuzisun That's right, so I also applied it to the install script (hack/generate-install.sh)

Reply:
I do not see it in `generate-install.sh`? also here can be removed as it won't work for 0.4.0

Reply:
@jazzsir actually this will not work as the v0.4.0 branch does not have the changes you made in master :(

Reply:
Let’s remove this instruction, we are releasing 0.5 soon anyway so we can generate the kubeflow manifest at that time. Otherwise this PR lgtm can be merged.

Reply:
I didn't come up with that, removed.